src/interfaces/ArrayManager.java

```diff
@@ -13,40 +13,35 @@
     /**
      * generate an array containing all letters for a number of alphabetMultiplier times
      */
-    public void setEvaluationArray();
+    void setEvaluationArray();
 
     /**
      * shuffle the array
      *
      * @param charArray - the array that needs to be shuffled
      */
-    public void shuffleCharArray(char[] charArray);
-
-    /**
-     * transfer the sorted chars to a char array
-     */
-    public void setCharArrayFromSortedMap();
+    void shuffleCharArray(char[] charArray);
```

Comment:
Inside `ManageArray.java`, please remove the line `this.shuffledArray = charArray;` from `public void shuffleCharArray(char[] charArray)` method.

Reply:
I don't understand. If I remove that line I would need to make the method return an array. Otherwise I need to assign the values to the shuffled array

Reply:
Yes, sir. That is the intent. ;)